wolfcrypt/src/asn.c

```diff
@@ -16806,6 +16811,16 @@ enum {
 #define certExtASN_Length (sizeof(certExtASN) / sizeof(ASNItem))
 #endif
 
+#if defined(WOLFSSL_CUSTOM_OID) && defined(WOLFSSL_ASN_TEMPLATE) \
+    && defined(HAVE_OID_DECODING)
+WOLFSSL_ASN_API void SetUnknownExtCallback(DecodedCert* cert,
```

Comment:
The `WOLFSSL_ASN_API` only goes in the header. Should have `int` return and please add `wc_` even though other ones don't have it.